Modules/_winapi.c

```diff
@@ -1347,15 +1347,15 @@ _winapi_PeekNamedPipe_impl(PyObject *module, HANDLE handle, int size)
         if (!ret) {
             return PyErr_SetExcFromWindowsErr(PyExc_OSError, 0);
         }
-        return Py_BuildValue("ii", navail, nleft);
+        return Py_BuildValue("II", navail, nleft);
     }
 }
 
 /*[clinic input]
 _winapi.ReadFile
 
     handle: HANDLE
-    size: int
+    size: DWORD
```

Comment:
Hmm... how come you changed this and the generated code didn't change? Did you run `make clinic` to update all files?

Reply:
no, sorry, I was not aware of the process and I thought that the files are regenerated on the build somehow, I committed the changes to the generated section

Reply:
the checks passed this time it seems, can you recheck if all good @pitrou?